frontend/src/cards/ui/MultiMapDialog.tsx

```diff
@@ -348,13 +339,15 @@ export function MultiMapDialog(props: MultiMapDialogProps) {
 
       {/* MODAL FOOTER */}
       <footer>
+        {/* mobile-only close button */}
```

Comment:
I think we should keep the "close" button in the footer for both mobile and desktop, and eliminate this top right "X" button entirely. do you agree? @kccrtv ?

<img width="981" alt="Screen Shot 2023-08-29 at 3 32 36 PM" src="https://github.com/SatcherInstitute/health-equity-tracker/assets/41567007/631eca0e-6b8c-4f7a-ab79-377789e2536c">

Reply:
Also, should we try to keep the sources and maybe even the notes if it multimap on desktop? only hide those on mobile?

Reply:
I forgot to add the mobile prop for the sources, fixing that now.